hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveViewCatalog.java

```diff
@@ -125,6 +125,51 @@ public void testHiveViewAndIcebergViewWithSameName() throws TException, IOExcept
         .hasMessageStartingWith("Not an iceberg view: hive.hivedb.test_hive_view");
   }
 
+  @Test
+  public void testHiveViewExists() throws IOException, TException {
+    String dbName = "hivedb";
+    Namespace ns = Namespace.of(dbName);
+    String viewName = "test_hive_view_exists";
+    TableIdentifier identifier = TableIdentifier.of(ns, viewName);
+    TableIdentifier invalidIdentifier = TableIdentifier.of(dbName, "invalid", viewName);
+    if (requiresNamespaceCreate()) {
+      catalog.createNamespace(identifier.namespace());
+    }
+
+    assertThat(catalog.viewExists(invalidIdentifier))
+        .as("Should return false on invalid view identifier")
+        .isFalse();
+    assertThat(catalog.viewExists(identifier)).as("View should not exist before create").isFalse();
+
+    catalog
+        .buildView(identifier)
+        .withSchema(SCHEMA)
+        .withDefaultNamespace(ns)
+        .withQuery("hive", "select * from hivedb.tbl")
+        .create();
+    assertThat(catalog.viewExists(identifier)).as("View should exist after create").isTrue();
+
+    catalog.dropView(identifier);
```

Comment:
Why did you deviate here from the table tests? Why not check the return value of the drop?